fix: pid/attestation deletion made by Holders must not notify to credential issuer (#860)
italia/eidas-it-wallet-docs: docs/en/credential-revocation.rst

Users MAY change their Digital Credential validity status by:

>   1. Deleting the Digital Credential from their Wallet Instance: the Wallet Instance SHOULD prompt the User about the deletion motivation to be optionally notified to the Credential Issuer. When the User uses this functionality, the notification to be sent to the Credential Issuer MUST use the Notification Endpoint provided by the Issuer, as described in Section :ref:`credential-revocation:Status Update by Wallet Instance`.

Reviewer: @vcarollo non mi torna questo tema della _reason_.  A mio parere, la motivazione dell'eliminazione locale è un dato personale non necessario per la finalità del trattamento (che è l'eliminazione del file locale o la revoca globale).

Author: rimosso, grazie @mikpozzo93

>   1. Deleting the Digital Credential from their Wallet Instance: the Wallet Instance SHOULD prompt the User about the deletion motivation to be optionally notified to the Credential Issuer. When the User uses this functionality, the notification to be sent to the Credential Issuer MUST use the Notification Endpoint provided by the Issuer, as described in Section :ref:`credential-revocation:Status Update by Wallet Instance`.

Author: ```suggestion
  1. Deleting the Digital Credential from their Wallet Instance: the Wallet Instance SHOULD prompt the User to be optionally notify the Credential Issuer about the User's intention to revoke the Digital Credential. When the User uses this functionality, the notification to be sent to the Credential Issuer MUST use the Notification Endpoint provided by the Issuer, as described in Section :ref:`credential-revocation:Status Update by Wallet Instance`.
```

Users MAY change their Digital Credential validity status by:

>   1. Using the Issuer's web portal:

Author: ```suggestion
  1. Deleting the Digital Credential from their Wallet Instance: the Wallet Instance SHOULD prompt the User about the deletion motivation to be optionally notified to the Credential Issuer. When the User uses this functionality, the notification to be sent to the Credential Issuer MUST use the Notification Endpoint provided by the Issuer, as described in Section :ref:`credential-revocation:Status Update by Wallet Instance`.
  2. Using the Issuer's web portal:
```

Status Update by Wallet Instance
""""""""""""""""""""""""""""""""

When the User deletes a Digital Credential from the Wallet Instance, the Wallet Instance by default SHALL NOT notify the Credential Issuer of this deletion event. Deleting a Digital Credential from the Wallet Instance only removes the local copy and does not change the validity status at the Issuer.

The Wallet Instance MAY inform the User, prior to deletion, that deletion is a local action and does not imply revocation at the Issuer, and MAY implement, under the User’s explicit consent at deletion time, a notification feature to inform the Credential Issuer of the User’s intention to revoke the Digital Credential.

> If the User wants the Issuer to revoke a Digital Credential, the User SHOULD explicitly confirm this intention via the Wallet Instance’s deletion prompt (when available), which SHALL then notify the Credential Issuer; alternatively, the User MAY use the Issuer's web portal or other Issuer-provided channels.

Author: ```suggestion
If the User wants the Issuer to revoke a Digital Credential, the User SHOULD explicitly confirm this intention via the Wallet Instance’s deletion prompt (when available), which SHALL then notify the Credential Issuer; alternatively, the User MAY use the Issuer's web portal or other Issuer-provided channels.

When the revoked Credential is the PID, the Credential Issuer MUST send a notification of this event to the User within 24 hours.
For any other Credential different from the PID, the Credential Issuer SHOULD send a notification of this event to the User. The notification to the User MAY be implemented in several ways, such as using a User's email address, telephone number, or any other verified and secure communication channel. The notification to the User MUST also include all the information about the Credential revocation status. The method used for the notification to the User is out of scope of the current technical implementation profile. When the revocation occurs, the Credential Issuer MUST update the status of the Digital Credential accordingly. When the Notification Response sent by the Credential Issuer is successfully received by the Wallet Instance, the Wallet Instance MUST delete the Digital Credential.
```

When multiple Digital Credentials are issued together in a single batch, their lifecycle remains fully granular:

>   * **Grouped triggers, independent updates**: A single batch status update request referencing the batch's ``notification_id`` and sent by an authorized entity (e.g. a Wallet Provider via PDND) is handled as N separate status changes. The Issuer updates each Credential's own status individually (for example, flipping its status-list bit to ``INVALID`` or ``SUSPENDED``). By default, a Wallet Instance SHALL NOT trigger batch status updates when the User deletes local Credentials. Upon deletion, the Wallet Instance MAY, under the User’s explicit consent, notify the Credential Issuer of the User’s intention to revoke the affected Credential(s); such a notification does not constitute a batch status update request.

Author: ```suggestion
  * **Grouped triggers, independent updates**: A single batch status update request referencing the batch's ``notification_id`` and sent by an authorized entity (e.g. the Wallet Instance via Notification Endpoint with ``event=credential_deleted``, a Wallet Provider via PDND) is handled as N separate status changes. The Issuer updates each Credential's own status individually (for example, flipping its status-list bit to ``INVALID`` or ``SUSPENDED``). By default, a Wallet Instance SHALL NOT trigger batch status updates when the User deletes local Credentials. Upon deletion, the Wallet Instance MAY, under the User’s explicit consent, notify the Credential Issuer of the User’s intention to revoke the affected Credential(s); such a notification does not constitute a batch status update request.
```
